internal/cmd/kafka/command_topic_consume_onprem.go

```diff
@@ -50,9 +50,8 @@ func (c *authenticatedTopicCommand) newConsumeCommandOnPrem() *cobra.Command {
 	cmd.Flags().Bool("timestamp", false, "Print message timestamp in milliseconds.")
 	cmd.Flags().String("delimiter", "\t", "The delimiter separating each key and value.")
 	cmd.Flags().StringSlice("config", nil, `A comma-separated list of configuration overrides ("key=value") for the consumer client.`)
-	cmd.Flags().String("config-file", "", "The path to the configuration file (in json or avro format) for the consumer client.")
+	cmd.Flags().String("config-file", "", "The path to the configuration file (in JSON or avro format) for the consumer client.")
```

Comment:
This flag is defined more than once. Can we write an `addConfigFile()` helper function?

Reply:
there're 4 kinds of different definitions of `config-file` flags in multiple commands. I'll do a clean-up pr to address all of them